code/socialDistribution/templates/tagtemplates/comment.html

```diff
@@ -7,18 +7,21 @@
             <div class="col col-sm-6 col-md-4 col-lg-3 d-flex align-items-center justify-content-center ml-2">
 
                 {% if comment.author.profileImageUrl %}
-                    <img src="{{ comment.author.profileImageUrl }}"
+                    <img src="{{ comment.author.profileImageUrl }}" onclick="redirectToAuhtorPage('{{author_type}}', '{{comment.author.id}}')"
```

Comment:
Would be better if we made this independent of remote/local authors. This can be left for a future PR, however

Reply:
sure, removing the local author check. we can come back to this and rethink the func when we support remote authors.  We can't be sure what frontend URL to redirect for remote authors so could redirect to the API route maybe.